client/src/components/Offer/OfferHelper.ts

```diff
@@ -0,0 +1,17 @@
+ 
+type Props = {
```

Comment:
Only for classes and components, we use PascalCase. Please adjust the file name. And also if it's only in one place used, it's not a helper, it could be just in the same file like the component that uses this.